src/js/jwt/jwt.js

```diff
@@ -287,6 +287,19 @@ function refreshTokens() {
   authChannel.postMessage({ type: 'refresh' });
 }
 
+const shouldPageAuth = (path) => {
+  if(path === '/') {
+    return false
+  }
+
+  const adjustedPath = path.replace(/\/$/, "");
+  if(allowedUnauthedPaths.includes(adjustedPath)) {
```

Comment:
I think we could just use this instead of the whole `if` statement.
```jsx
return allowedUnauthedPaths.includes(path.replace(/\/$/, ""))
```

Reply:
I don't think this will work for the '/' page?

Reply:
I meant for the second if

Reply:
to get rid of the `if(x) {return false} else {return true}` we can just `return !x`.